src/com/jsyn/engine/SynthesisEngine.java

```diff
@@ -233,7 +233,9 @@ public synchronized void start(int frameRate, int inputDeviceID, int numInputCha
             engineThread = new EngineThread(inputDeviceID, numInputChannels,
                     outputDeviceID, numOutputChannels);
             logger.fine("Synth thread old priority = " + engineThread.getPriority());
-            engineThread.setPriority(engineThread.getPriority() + 2);
+            int engineThreadPriority = engineThread.getPriority() + 2 > Thread.MAX_PRIORITY ?
```

Comment:
This uses the constant 2 in two places. This could lead to a bug where one is changed but not the other. This is safer:

int engineThreadPriority =  Math.min(engineThread.getPriority() + 2, Thread.MAX_PRIORITY);